### ISSUE TITLE
XL subs have extreme issues with status monitor and sonar

### ISSUE THREAD

**Issue by MudraptorSeks** [Regalis11/Barotrauma#7451]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
1- the sonar monitor zooms out based on a set scale, not based on submarine size. therefore XL subs get extremely small range extending off the submarine.
2. the status monitor cannot properly display rooms. it is far too zoomed out and many rooms overlap. perhaps adding a way to zoom in to the status monitor, similar to sub preview would be a good solution

**Steps To Reproduce**
have very large sub (bigger than the olympus). open status and sonar. try to function.
**Version**
Most modern unstable build at time of post
**Additional information**
Please I wanna make HUGE subs and other than this it works great

**Comment by Regalis11** [comment]
Thank you for the report!

We have an existing ticket about the 1st issue (https://github.com/Regalis11/Barotrauma/issues/6990), but the 2nd one is something we need to look into as well.

**Comment by MudraptorSeks** [comment]
> Thank you for the report!
> 
> We have an existing ticket about the 1st issue (#6990), but the 2nd one is something we need to look into as well.

Ok, cool! Didn't find anything but apparently I suck at searching. They did a better job explaining anyway

**Comment by Ek-Videogames** [comment]
So basically a 'zoom' feature much like what we already for the electrical-display and item-display, just for hulls. Makes sense to me.

**Comment by itchyOwl** [comment]
Is this still an issue?

**Comment by Regalis11** [comment]
Yes, we still haven't done anything about this. Imo we should make the view zoomable and moveable.
